Plash/URL+Placeholder.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  URL+Placeholder.swift
+//  Plash
+//
+//  Created by Jonathan Cole on 8/31/20.
+//  Copyright © 2020 Sindre Sorhus. All rights reserved.
+//
+
+import Foundation
+
+enum URLPlaceholderError: Error, LocalizedError {
```

Comment:
This could be nested inside the `URL` extension and renamed to `PlaceholderError`.

Reply:
👍